### PR TITLE
Add Limit warnings and smoke (S) for aerobatics

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
last commit closes #70

File: ask21-set.xml [hunk 1]
```diff
@@ -368,6 +371,22 @@ See AUTHORS.txt for credits
             </winch>
         </hitches>
     </sim>
+    <limits>
+        <max-spoiler-extension-speed>135</max-spoiler-extension-speed>
```
Comment (viktorradnai) [inline ask21-set.xml]: This is wrong, according to the flight manual, max spoiler extension speed is the same as Vne, 280km/h. On the other hand the max G force the aircraft can withstand is reduced with the airbrakes open.

Comment (viktorradnai) [reply ask21-set.xml]: See here on page 10: https://www.capmembers.com/media/cms/ASK21__Schleicher__Flight_Manual_0F12AEE54A4A2.pdf

**Review by viktorradnai — COMMENTED** [review]
We might want to do a negative airspeed check as well. Since tailslides are forbidden, above a certain speed (30-40 knots?) we could probably detach the tailplane or at least warn about it.

File: ask21-set.xml (hunk 1 above)
Comment (ghost) [reply ask21-set.xml]: I was referring to http://www.segelfluggruppe-isartal.de/der-verein/die-flotte/ask21/ ("Sturzflugbremsen"), but probably a type error there ;)

Comment (ghost) [reply ask21-set.xml]: Affirmative, should be 151.2kts according to PoH

File: ask21-set.xml [hunk 2]
```diff
@@ -504,13 +524,22 @@ See AUTHORS.txt for credits
                     <min>0</min>
                 </binding>
             </key>
+            <key n="83">
+                <name>S</name>
```
Comment (viktorradnai) [inline ask21-set.xml]: The 'S' key is already used to increment the variomenter volume (s/S turns it up and down). Please choose another key (maybe ctrl-s?). The smoke looks great btw.

Comment (ghost) [reply ask21-set.xml]: Oops :D will do with Ctrl-s

**Comment by ghost** [comment]
I also re-rendered effect textures for new model

**Review by viktorradnai — APPROVED** [review]
Looks good now, although I'm not sure if we need the Vno warning.

**Pull request merged**